plugins/GSdx/GSdx.rc

```diff
@@ -236,6 +236,8 @@ BEGIN
     COMBOBOX        IDC_ADAPTER,71,55,166,118,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
     LTEXT           "Renderer:",IDC_STATIC,6,72,34,8
     COMBOBOX        IDC_RENDERER,71,70,166,118,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    COMBOBOX        IDC_MIPMAP,71,100,166,118,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    LTEXT           "Mipmap (Insert):",IDC_MIPMAP_TEXT,6,102,53,8
```

Comment:
It was taken from [gregory's code](https://github.com/PCSX2/pcsx2/commit/3c30db965dfcc41f101e43ede37980506630e17d#diff-748d76fbcaa43637145d131d37054679R397), so it's all in the hands of @gregory38 if he wants to change it :)

Personally I just want the Linux-Windows GUI's to have similar names.